### ISSUE TITLE
UG1 font

### ISSUE THREAD

**Issue by JOHN30011887** [xan1242/NFS_CustomFont#2]
Iv noticed that on the loading screens in Underground 1, the font doesn't fit the whole width of the screen anymore, its all crammed in the middle and often getting cut off the bottom
![speed_2022_11_17_00_57_47_177](https://user-images.githubusercontent.com/12979984/202328412-ff1e3f20-51bb-458a-9d1f-2a5e243e3a0a.png)

This is how it looks without this mod
![speed_2022_11_17_01_06_08_326](https://user-images.githubusercontent.com/12979984/202329070-981b802e-d9dd-453d-9531-8a6e1359f31f.png)

My game res is 1080p if that means anything to help
All other font with this mod seem to be perfect

**Comment by xan1242** [comment]
Yeah it seems I missed the loading screen because I don't actually use this loading screen in my build...

Will check soon...

**Comment by Bogeudan** [comment]
![21436D52-9755-41A0-84CB-7110F9802121](https://github.com/xan1242/NFS_CustomFont/assets/158072704/b633f118-a010-4302-a4b9-aa7c182de3ad)
![9794066C-C03C-4F50-B878-B65403E471E0](https://github.com/xan1242/NFS_CustomFont/assets/158072704/2edc37ec-41fb-47c1-8a12-3c3600fb2115)
Same in the EA Trax Menu and Message Boxes
